x/pairing/keeper/grpc_query_user_entry.go

```diff
@@ -31,9 +31,33 @@ func (k Keeper) UserEntry(goCtx context.Context, req *types.QueryUserEntryReques
 	// support legacy
 	project, vrfpk_proj, err := k.GetProjectData(ctx, userAddr, req.ChainID, epochStart)
 	if err == nil {
-		allowedCU := project.Policy.EpochCuLimit
+		plan, err := k.subscriptionKeeper.GetPlanFromSubscription(ctx, project.GetSubscription())
+		if err != nil {
+			return nil, err
+		}
+
+		planPolicy := plan.GetPlanPolicy()
+
+		err = project.VerifyProject(req.ChainID, planPolicy)
```

Comment:
we need to verify with the other policies as well

Reply:
inside the VerifyProject function I also check the other policies (it's called with the planPolicy alone because the other policies are fields inside the project, so I don't need to get them as arguments since VerifyProject has a project receiver)